### PR TITLE
Update fluent-ffmpeg for es6 module library import 

### PR DESCRIPTION
**Pull request by jgainfort**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [n/a] Provide a URL to documentation or source code which provides context for the suggested changes: 
- [n/a] Increase the version number in the header if appropriate.
- [n/a] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dslint/dt.json" }`.

The declaration file was almost unusable and required a very minor tweak.

Originally the test file used a UMD import style of
`import * as ffmpeg from 'fluent-ffmpeg'`

This type of import would throw an error in node environment
screen shot 2017-05-11 at 11 01 37 am

Thus an import of `import ffmpeg = require('fluent-ffmpeg')` was required

Under typescript handbook this library more suited a [module library](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html) and required an export as namespace options which can be found [here](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html)

Now this allows me to do stuff like

```
import ffmpeg = require('fluent-ffmpeg');

class FfmpegAdaptor {
    private ffmpegCommand: Ffmpeg.FfmpegCommand;

    constructor(options) {
        this.ffmpegCommand = ffmpeg(options)
        ...
    }

    ....
}
```

### CODE REVIEW

**Comment by jgainfort** [comment]
@DingWeizhe Im not sure if you are the active maintainer. I cant seem to @tcaesvk. I would love your thoughts.

**Comment by aozgaa** [comment]
@jgainfort can you edit the test so that it uses the sample code on the [project page](https://github.com/fluent-ffmpeg/node-fluent-ffmpeg)?

**Comment by jgainfort** [comment]
@aozgaa There are a lot of samples on the project page so I only added a couple. That alright?

**Comment by aozgaa** [comment]
Yup, that's fine. Just good to have some sanity checks in place.

Looks like tests are failing, please fix the type definitions to work with the tests.

**Comment by jgainfort** [comment]
@aozgaa Oops, sorry. Fixed them.

**Comment by jgainfort** [comment]
@aozgaa Any chance this can be merged?

File: types/fluent-ffmpeg/tsconfig.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
 {
-    "compilerOptions": {
```
Comment (andy-ms) [inline types/fluent-ffmpeg/tsconfig.json]: tsconfigs are formatted automatically by `scripts/generate-tsconfigs.ts`, don't change it manually.

File: types/fluent-ffmpeg/index.d.ts [hunk 2]
```diff
@@ -296,9 +297,12 @@ declare namespace Ffmpeg {
...
         concatenate(target: string | stream.Writable, options?: { end?: boolean }): FfmpegCommand;
         concat(target: string | stream.Writable, options?: { end?: boolean }): FfmpegCommand;
+        clone(): FfmpegCommand;
+        run(): void;
     }
 }
 declare function Ffmpeg(options?: Ffmpeg.FfmpegCommandOptions): Ffmpeg.FfmpegCommand;
 declare function Ffmpeg(input?: string | stream.Readable, options?: Ffmpeg.FfmpegCommandOptions): Ffmpeg.FfmpegCommand;
 
+export as namespace Ffmpeg;
```
Comment (andy-ms) [inline types/fluent-ffmpeg/index.d.ts]: This should only be here if the package creates an `Ffmpeg` global variable. Can you link to documentation for this?

File: types/fluent-ffmpeg/tsconfig.json (hunk 1 above)
Comment (jgainfort) [inline types/fluent-ffmpeg/tsconfig.json]: that just might be my editor doing an auto format. Doesn't look like anything really changed other than spacing.

File: types/fluent-ffmpeg/index.d.ts (hunk 2 above)
Comment (jgainfort) [inline types/fluent-ffmpeg/index.d.ts]: If that line is not included I am unable to get any types from the definition. Lib is used in node.js app and is imported by

`import ffmpeg = require('fluent-ffmpeg')`

If you have a better suggestion I can modify.

Comment (andy-ms) [inline types/fluent-ffmpeg/index.d.ts]: The following works with the current type definitions:
```ts
import ffmpeg = require("fluent-ffmpeg");
const x: ffmpeg.Encoders = {};
```

Comment (jgainfort) [inline types/fluent-ffmpeg/index.d.ts]: Ok, I will remove

**Pull request merged**